src/App.tsx

```diff
@@ -1,26 +1,56 @@
-import React, { useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './helpers/api';
 
 const App: React.FC = () => {
   const [
     selectedUserId,
-    // setSelectedUserId,
+    setSelectedUserId,
   ] = useState(0);
 
+  const [todos, setTodos] = useState<Todo[]>([]);
+
+  const selectedUserChangeHandler = useCallback(
+    (newSelectedUserId: number) => setSelectedUserId(newSelectedUserId),
+    [],
+  );
```

Comment:
but why? you only have to use useCallback with custom callbacks, no need to do it got functions that are returned by useState